anonymity.md

```diff
@@ -0,0 +1,52 @@
+# Anonymity on [Mapknitter](https://mapknitter.org)
+
+[Publiclab](https://publiclab.org) is an organization dedicated to protecting the planet by building useful [tools](https://github.com/publiclab) and bringing together passionate and kind people who want to have a positive impact. Mapknitter was at the beginning of this collaboration between committed individuals when it was used to [provide public information about the 2010 BP Oil Disaster](https://publiclab.org/wiki/stories). It has since grown and now hosts more than **2761** maps yet it still upholds its fundamental idea of sharing **open** data that anyone can **contribute to or edit**.  
+
+Users **should not** be bogged down with privacy issues and the extra barrier of creating an account as they advance this initiative but fighting the [problems of anonymous contributions](https://github.com/publiclab/mapknitter/issues/1029) is also important.
+
+## Privacy and Impact
+ Supporting **user privacy** is crucial to Mapknitter and its contributors because of this information explained by @jywarren:
```

Comment:
```suggestion
 Supporting **user privacy** is crucial to MapKnitter and its contributors because of this information explained by [@jywarren](https://github.com/jywarren):
```